src/clip.sol

```diff
@@ -326,7 +326,10 @@ contract Clipper is LibNote {
     }
 
     // Cancel an auction during ES
-    function yank() external note auth {
-        // TODO
+    function yank(uint id) external auth {
```

Comment:
IMO we should go ahead and call `_remove(id)` here, since we might call this function as part of a spell after a circuit breaker trip, and otherwise we can't keep the list of active ids consistent.

Reply:
Fixed